test/unit/expressions/expressionCombinedTransformations-test.js

```diff
@@ -192,7 +192,7 @@ describe('Combine Jexl and legacy expressions (default JEXL)', function () {
                 .matchHeader('fiware-servicepath', 'gardens')
```

Comment:
We have also a variant of this tests for ngsiv2

test/unit/ngsiv2/expressions/jexlBasedTransformations-test.js

That file doesn't need modifications? (looking to CI results, it seems it doesn't, but I wonder why...)

Reply:
expressionCombinedTransformations-test.js includes tests which legacy and jexl expression, but not any of now not allowed cases:

    Update for an integer attribute with string expression
    Update for a Float attribute with string expression
    Update for a Null attribute with arithmetic expression
    Update for a Null attribute with string expression
    Update for a Boolean attribute with arithmetic expression
    Update for a Boolean attribute with string expression

Reply:
I have had a look into this with more detail.

In test/unit/expressions/expressionCombinedTransformations-test.js (NGSIv1 variant) the only difference between old files (e.g. test/unit/examples/contextRequests/updateContextExpressionPlugin3.json) and new files (test/unit/examples/contextRequests/updateContextExpressionPlugin8.json) is that the old uses `"value": "1040"` ("stringfied") and the new `"value": 1040` ("native").

On the other hand test/unit/ngsiv2/expressions/expressionCombinedTransformations-test.js (NGSIv2 variant) already uses the native way, e.g. see test/unit/ngsiv2/examples/contextRequests/updateContextExpressionPlugin1.json.

Thus, I think that explains that only test/unit/expressions/expressionCombinedTransformations-test.js is changed in this PR but not test/unit/ngsiv2/expressions/expressionCombinedTransformations-test.js

NTC